qa/rpc-tests/sc_csw_eviction_from_mempool.py

```diff
@@ -0,0 +1,299 @@
+#!/usr/bin/env python2
+# Copyright (c) 2014 The Bitcoin Core developers
+# Copyright (c) 2018 The Zencash developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.authproxy import JSONRPCException
+from test_framework.util import assert_equal, assert_true, assert_false, initialize_chain_clean, \
+    stop_nodes, wait_bitcoinds, \
+    start_nodes, sync_blocks, sync_mempools, connect_nodes_bi, disconnect_nodes, mark_logs, \
+    dump_sc_info_record, get_epoch_data, swap_bytes, advance_epoch
+from test_framework.mc_test.mc_test import *
```

Comment:
I would avoid importing unused symbols.

```suggestion
from test_framework.mc_test.mc_test import CertTestUtils, CSWTestUtils, generate_random_field_element_hex
```

Reply:
Done